R/bootstrap.nlsfit.R

```diff
@@ -157,16 +159,16 @@ parametric.nlsfit.cov <- function (fn, par.guess, boot.R, y, x, cov, ..., bootst
   if (bootstrap) {
     stopifnot(!missing(boot.R))
     bsamples <- parametric.bootstrap.cov(boot.R, values, cov)
-    bootstrap.nlsfit(fn, par.guess, y, x, bsamples, ..., CovMatrix = cov)
+    bootstrap.nlsfit(fn, par.guess, y, x, bsamples, ..., CovMatrix = cov, na.rm)
   }else {
     if(missing(boot.R)) {
       boot.R = 0
     }
-    simple.nlsfit(fn, par.guess, y, x, errormodel, ..., CovMatrix = cov, boot.R = boot.R)
+    simple.nlsfit(fn, par.guess, y, x, errormodel, ..., CovMatrix = cov, boot.R = boot.R, na.rm)
   }
 }
 
-get.errors <- function (useCov, y, dy, dx, CovMatrix, errormodel, bsamples, cov_fn, error) {
+get.errors <- function (useCov, y, dy, dx, CovMatrix, errormodel, bsamples, cov.fn, error) {
```

Comment:
why the change of notation for `cov_fn`? This makes it inconsistent with the rest of hadron

Reply:
@j-ostmeyer has argued at some point that it should not be inconsistent within the arguments of a single function. But we are actually well past this point with `useCov`, `CovMatrix`, `cov_fn` and `use.minpack.lm`. Modern R (I guess as defined by Hadley Wickham) uses underscores, and that's what I would aim for in the long run.